views/add_event.hbs

```diff
@@ -20,19 +20,18 @@
     <label>End Time: </label>
     <input class="form-control" type="text" name="endTime" placeholder="example 00:00" required>
 
-    <label>Place: </label>
+    <label>Location: </label>
     <li id="addPlaceIcon" class="glyphicon glyphicon-plus"></li>
-    <span class="addPlaceSpan text-muted">add new place</span>
+    <span class="addPlaceSpan text-muted">add new location</span>
```

Comment:
I would keep this as 'add new place'

Reply:
i'm not sure about place doesn't sound as good as location 
what do you think @JWLD ?

Reply:
The thing is, 'places' have a 'location' field. So as it is, you would add a new location, then have a field for the location of the location